database/seeders/_user.js

```diff
@@ -0,0 +1,7 @@
+const { adminUser, adminUser2 } = require('../../constants/mocks');
```

Comment:
Why the use of underscore here `_user.js `?

Reply:
The reason for the underscore is to enable sequelize seed user data and role data before userRole data.__